bindings/python/iota_sdk/types/transaction_data.py

```diff
@@ -34,7 +34,7 @@ class RemainderData:
         chain: The BIP44 chain for the remainder address.
     """
     output: Output
-    address: Address
+    address: Ed25519Address | AliasAddress | NFTAddress
```

Comment:
Why ?

Reply:
Doesn't work otherwise

Reply:
What's the difference between output and address? Why can output be the enum and not address?

Reply:
Forgot to update the Output there